### ISSUE TITLE
Search tabs in high contrast mode

### ISSUE THREAD

**Issue by agbeltran** [ral-facilities/datagateway#1005]
**Description**:
When in high contrast mode, the selected search tab is not clear, as the colour used for selection is the same as the background colour of the table header.

<img width="1542" alt="Screenshot 2021-12-14 at 11 22 33" src="https://user-images.githubusercontent.com/1417033/145989261-4647b04c-87be-46b7-b837-ad110e652c40.png">

<img width="1550" alt="Screenshot 2021-12-14 at 11 22 46" src="https://user-images.githubusercontent.com/1417033/145989264-20540fce-1b50-40d0-afb9-9c2031bf28d7.png">

**Acceptance criteria**:
- [ ] Change colour of selection in search tabs when using high contrast mode

**Comment by joelvdavies** [comment]
Just had a look, the tabs should actually have a tab indicator but it wasn't working correctly, I have just fixed that so it now looks like
![image](https://user-images.githubusercontent.com/90245114/146222518-9490361b-c807-495b-99ed-46ffa196fd18.png)
 There was a contrast issue previously that forced it to be a darker text colour so I can't really change that.

**Comment by joelvdavies** [comment]
Turns out the previous behaviour was needed for download, so this issue will need a more specific solution in datagateway instead.

**Comment by agbeltran** [comment]
hi, @joelvdavies, yes, the tab indicator works well in the other modes (dark mode, light mode) but it is not visible on the high-contrast mode.
